VDA-XP-Reward/README.md

```diff
@@ -0,0 +1,67 @@
+# Contract Explain
+## Description
+This contract receive signed off chain data (including Verida XP points) and use that to provide a reward to the address in the input parameters. The contract owner set and update XP to token conversion rate.
+
+The key objectives are:
+
+1. Every month, VDA is deposited into the smart contract as reward tokens
+2. Every month, Verida users can provide a proof of a signed data record containing XP points
+3. Data will be signed by a trusted Verida whitelisted DID
+4. Users can claim more than one XP proof at a time
+5. XP proofs are generated in one month, and can only be claimed in the following month
+6. Users that don't claim their XP in the correct month, forfeight their XP rewards
+7. Each month will have a different XP to VDA conversion rate
+8. Only the contract owner can change the XP to VDA conversion rate
+9. Any unclaimed VDA will be included in the next month reward pool
+10. Each proof can only be used once
+
+__*Reference*__
+
+https://github.com/verida/blockchain-contracts/issues/152
+
+## Dependencies
+### Verida contracts
+This contract dependes on the followig Verida contract:
+- `@verida/vda-verification-contract` : VDA-Verification-Base contract
```

Comment:
Explain why each of these dependencies are required

Reply:
Updated. Pls check again.

Reply:
Thanks.

You can remove `did-registry-contract` now?

Reply:
Updated.